test/e2e/storage/regional_pd.go

```diff
@@ -229,7 +229,9 @@ func testZonalFailover(c clientset.Interface, ns string) {
 	err = waitForStatefulSetReplicasReady(statefulSet.Name, ns, c, framework.Poll, statefulSetReadyTimeout)
 	if err != nil {
 		pod := getPod(c, ns, regionalPDLabels)
-		framework.ExpectEqual(podutil.IsPodReadyConditionTrue(pod.Status), true, "The statefulset pod has the following conditions: %s", pod.Status.Conditions)
+		if !podutil.IsPodReadyConditionTrue(pod.Status) {
+			framework.Failf("The statefulset pod %s has the following conditions: %v", pod.Name, pod.Status.Conditions)
```

Comment:
Can you add to the message here that the pod was expected to be ready?

Reply:
done